theano/compile/function_module.py

```diff
@@ -751,7 +752,13 @@ def checkSV(sv_ori, sv_rpl):
         f_cpy.maker.fgraph.name = name
         return f_cpy
 
-    def __call__(self, *args, **kwargs):
+    def apply(self, inputs, outputs):
+        # TODO: assert that there's a list of outputs
+        self.output_idx = outputs # indirectly passing output indices
+        # assuming for now I pass all the inputs
+        return [self.__call__(*inputs)[i] for i in outputs]
+
+    def __call__(self, *args, **kwargs): # sygi: what those kwargs mean?
```

Comment:
Thanks for the answer. I found the usage of `n_calls`, but it is on the `fn` attribute of the `Function` instance (`theano_conv.fn(n_calls=n_calls)`), and here is the `Function` instance itself. I am asking about that, because I'd prefer to pass the output_idx through the kwarg, but there's an assert later that prevents me from doing that.